model/spknet.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+"""
+    Build the LSTM(BLSTM)  neural networks for speaker recognition.
+
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import sys
+import time
+
+import tensorflow as tf
+from tensorflow.contrib.rnn.python.ops import rnn
+import numpy as np
+
+class LSTM(object):
+    """Build BLSTM or LSTM model for speaker recognition.
+       If you use this module to train your module, make sure that 
+       your prepare the right format data! 
+ 
+    Attributes:
+        config: Used to config our model
+                config.input_size: feature (input) size;
+                config.output_size: the final layer(output layer) size;
+                config.rnn_size: the rnn cells' number
+                config.batch_size: the batch_size for training
+                config.rnn_num_layers: the rnn layers numbers
+                config.keep_prob: the dropout rate
+        inputs_spk1: the speech feature of speaker1
+        inputs_spk2: the speech feature of speaker2
+        labels: "two hot" target label
+        infer: bool, if training(false) or test (true)
+    """
+
+    def __init__(self, config, inputs_spk1,inputs_spk2, labels, lengths, infer=False):
```

Comment:
I think we should process the combination of spk1 and spk2 outside the LSTM.  LSTM(.....) only accepts the spker feature matrix. 

def __init__(self, config, spk_feats, labels, lenghts, infer=False)

Reply:
all right, will do